spec/quality_spec.rb

```diff
@@ -178,6 +178,7 @@ def check_for_specific_pronouns(filename)
       console_command
       deployment_means_frozen
       forget_cli_options
+      github_https
```

Comment:
You didn't apply my suggestion of changing `github_https` to `github.https` in the list of settings, and instead removed it from the list. So you should now remove it from the list of exemptions as well.